tchannel/zipkin/zipkin_trace.py

```diff
@@ -29,13 +29,17 @@
 class ZipkinTraceHook(EventHook):
     """generate zipkin-style span for tracing"""
 
-    def __init__(self, tchannel=None, dst=None):
+    DEFAULT_RATE = 0.01
+
+    def __init__(self, tchannel=None, dst=None, sample_rate=None):
         """Log zipkin style trace.
 
         :param tchannel:
             The tchannel instance to send zipkin trace spans
         :param dst:
             The destination to output trace information
+        :param sample_rate:
+            The rate of sampling is in the range (0, 1] with 0.01 precision.
```

Comment:
Documentation should mention what the field is/how it affects behavior, not just the valid values.